### PR TITLE
Adjust line-height of route link to avoid clipping

### LINKED ISSUE #5270: Descenders clipped in route link on overview page
See screenshots. The `j` and `g` are cut off.

openshift_web_console

/cc @jwforres @sg00dwin

### PR DESCRIPTION
**Pull request by spadgett**
openshift_web_console

Fixes #5270

### CODE REVIEW

**Comment by spadgett** [comment]
@jwforres ptal

File: assets/app/styles/_components.less [hunk 1]
```diff
@@ -47,6 +47,10 @@
     border: 1px solid #ddd;
     position: relative;
     .osc-object-highlight();
+    h2 {
+      // Avoid clipping descenders in routes on overview.
+      line-height: 1.3;
```
Comment (jwforres) [inline assets/app/styles/_components.less]: surprised the line-height for the h2 wasnt correct, was something overriding the base styles?  maybe we need to fix our _variables.less file if not

Comment (spadgett) [inline assets/app/styles/_components.less]: @jwforres it's coming from patternfly.

```
patternfly/components/bootstrap/less/variables.less
70:@headings-line-height:    1.1;
```

Comment (spadgett) [inline assets/app/styles/_components.less]: Looks like the `overflow: hidden;` style in conjunction with the patternfly line height is causing the clipping, but we need `overflow` set for `text-overflow: ellipses;`

**Comment by jwforres** [comment]
[merge]

**Pull request merged**